generators/wp/templates/chisel-starter-theme/Chisel/Media.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Chisel;
+
+/**
+ * Class Media
+ * @package Chisel
+ *
+ * Default media settings for Chisel
+ */
+class Media {
+
+	public function __construct() {
+		// $this->addImageSizes();
```

Comment:
I'd uncomment this one and add some default image sizes just to start with

Reply:
I think we should also add a note in readme, that adding image sizes and using it in Image.src() method is a good practice, because not everywhere you need a biggest image shown, and you do not need to load a huge, heavy image and scale it down to show the thumbnail. Wordpress can make it smaller and save some megs on site load